src/components/Security/Roles/Filters.vue

```diff
@@ -29,27 +41,47 @@
 </template>
 
 <script>
+import { mapGetters } from 'vuex'
+
 export default {
   name: 'RolesFilters',
   components: {},
   props: {
     currentFilter: Object
   },
+  computed: {
+    ...mapGetters('kuzzle', ['$kuzzle'])
+  },
   data() {
     return {
-      controllers: []
+      controllers: [],
+      availableControllers: []
     }
   },
   methods: {
     resetSearch() {
       this.controllers = []
+    },
+    addControllerTag(controller) {
+      if (this.controllers.includes(controller)) {
+        return
+      }
+      this.controllers.push(controller)
+    },
+    async getKuzzlePublicApi() {
+      const publicApi = await this.$kuzzle.query({
```

Comment:
what happens if the connected user does not have the rights on this action or simply if the request fails?